mesheryctl/internal/cli/root/models/view.go

```diff
@@ -0,0 +1,127 @@
+package model
```

Comment:
Make sure if you're naming a package, it must match the name of directory too. If it's `model` please rename the directory to `model` and not `models`

Reply:
Noted.